### DISCUSSION TITLE
Update YAML data from a list

### DISCUSSION THREAD

**Discussion by mrfusiontm** [marcusolsson/obsidian-projects#594] [General]
I *LOVE* the Boolean toggle ( as can be seen in the 4K column in the image), and I'm wondering if it would be possible to select yaml data from a pulldown list.  The "Quality" data should only be "Lo", "Med" or "Hi"  and I'd much rather select an option from a pull down rather than typing the same three options over and over or perhaps choose an option with radio buttons that only allow one option to be selected. 
![Screen Shot 2023-06-23 at 9 07 40 AM](https://github.com/marcusolsson/obsidian-projects/assets/6153579/aec6c7e9-3c55-4a80-a48c-47a2b8c52f0d)

**Comment by marcusolsson** [comment]
If you click the three dots in the column header, you can configure the field to add options:

<img width="572" alt="Screenshot 2023-06-26 at 10 56 34" src="https://github.com/marcusolsson/obsidian-projects/assets/8396880/580c2b19-7a77-45b8-82f7-2211b8186e23">
<img width="218" alt="Screenshot 2023-06-26 at 10 56 50" src="https://github.com/marcusolsson/obsidian-projects/assets/8396880/cba13dee-8466-4947-9d2f-4112b26ef0c4">

However, this is more of a auto-completion rather than a dropdown.

**Reply by mrfusiontm** [reply]
Awesome, I'll check that out!  Thanks!
